src/commands/tokens/send_ft/mod.rs

```diff
@@ -57,3 +64,113 @@ impl SendFtCommand {
         )
     }
 }
+
+#[allow(clippy::too_many_arguments)]
+#[tracing::instrument(
+    name = "Creating a pre-populated transaction for signature ...",
+    skip_all
+)]
+pub fn get_prepopulated_transaction(
+    network_config: &crate::config::NetworkConfig,
+    ft_contract_account_id: &near_primitives::types::AccountId,
+    receiver_account_id: &near_primitives::types::AccountId,
+    signer_id: &near_primitives::types::AccountId,
+    amount_ft: &crate::types::ft_properties::FungibleToken,
+    memo: &Option<String>,
+    deposit: &crate::types::near_token::NearToken,
+    gas: &crate::common::NearGas,
+) -> color_eyre::eyre::Result<crate::commands::PrepopulatedTransaction> {
+    let args = serde_json::to_vec(&json!({
+        "receiver_id": amount_ft.amount().to_string(),
+        "amount": amount_ft.amount().to_string(),
+        "memo": memo.as_ref().and_then(|s| {
+            let trimmed = s.trim();
+            if trimmed.is_empty() {
+                None
+            } else {
+                Some(trimmed.to_string())
+            }
+        })
+    }))?;
+
+    let action_ft_transfer = near_primitives::transaction::Action::FunctionCall(Box::new(
+        near_primitives::transaction::FunctionCallAction {
+            method_name: "ft_transfer".to_string(),
+            args,
+            gas: gas.as_gas(),
+            deposit: deposit.as_yoctonear(),
+        },
+    ));
+
+    let args = serde_json::to_vec(&json!({"account_id": receiver_account_id.to_string()}))?;
+
+    let call_result = network_config
+        .json_rpc_client()
+        .blocking_call_view_function(
+            ft_contract_account_id,
+            "storage_balance_of",
+            args.clone(),
+            near_primitives::types::Finality::Final.into(),
+        )
+        .wrap_err_with(||{
+            format!("Failed to fetch query for view method: 'storage_balance_of' (contract <{}> on network <{}>)",
+                ft_contract_account_id,
+                network_config.network_name
+            )
+        })?;
+
+    if call_result.parse_result_from_json::<Value>()?.is_null() {
+        let action_storage_deposit = near_primitives::transaction::Action::FunctionCall(Box::new(
+            near_primitives::transaction::FunctionCallAction {
+                method_name: "storage_deposit".to_string(),
+                args,
+                gas: gas.as_gas(),
+                deposit: near_token::NearToken::from_millinear(100).as_yoctonear(),
+            },
+        ));
+        return Ok(crate::commands::PrepopulatedTransaction {
+            signer_id: signer_id.clone(),
+            receiver_id: ft_contract_account_id.clone(),
+            actions: vec![action_storage_deposit, action_ft_transfer.clone()],
+        });
+    }
+
+    Ok(crate::commands::PrepopulatedTransaction {
+        signer_id: signer_id.clone(),
+        receiver_id: ft_contract_account_id.clone(),
+        actions: vec![action_ft_transfer.clone()],
```

Comment:
Try to drop these `.clone()`, I feel it should work just fine.

```suggestion
            actions: vec![action_storage_deposit, action_ft_transfer],
        });
    }

    Ok(crate::commands::PrepopulatedTransaction {
        signer_id: signer_id.clone(),
        receiver_id: ft_contract_account_id.clone(),
        actions: vec![action_ft_transfer],
```